packages/edit-post/src/components/init-pattern-modal/index.js

```diff
@@ -74,6 +74,7 @@ export default function InitPatternModal() {
 								__next40pxDefaultSize
 							/>
 							<ToggleControl
+								__nextHasNoMarginBottom
```

Comment:
## Testing instructions

Go to `wp-admin/post-new.php?post_type=wp_block`.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/016e8827-8fa5-4e35-b451-70c15bb0acf2" alt="Create pattern modal, before" width="370">|<img src="https://github.com/user-attachments/assets/02b39761-fbab-4de8-9ce0-b25904cd92db" alt="Create pattern modal, after" width="373">|

Reply:
[Related discussion](https://github.com/WordPress/gutenberg/pull/64212#discussion_r1702224726) about how much space we actually want before the primary button. Tl;dr — we're not sure yet.